.github/workflows/weekly_run.yml

```diff
@@ -17,7 +17,11 @@ jobs:
       - name: Set up Ruby
         uses: ruby/setup-ruby@v1
         with:
-          ruby-version: 3.3.4
+          ruby-version: 2.5
+
+      - name: Install mechanize gem
+        run: |
+          gem install mechanize
```

Comment:
run bundle install ?

Reply:
bundle is unncessary no ? 
gem is installing on its own  but ill try with bundle in the coming commits

Reply:
beacuse run bundle install installs all the libs in the gemfile 
and we only need mechanize